EngineLayer/Mods/missing.txt

```diff
@@ -1,126 +1,147 @@
 Missing
-ID   W
+ID   Missing Tryptophan (W)
```

Comment:
Clarifies that these are missing. This is what I'm using in ProteoformSuite.

If you like the conciseness of before, maybe you could at least put "-W" or "W-1" instead of "W."